Update team info. (#61)
vesoft-inc/nebula-community: hackathon/hackathon-2021/teams.md

* Team members
   *  Hao Wen
   *  Mingquan Ji

### Intelligent Learning

* Project name: Learning assistant
* Project Introduction: Focus on intelligent knowledge management, intelligent documents and assistants to develop a new generation of knowledge carriers, simplifying the knowledge and promoting learning innovation!
    * Zhenhua Zhao
    * Youshuai Wang
    * Sun
    * Wushang
  
## Team members recruiting

### Pingan Technology

* Details of the team
   * Team name
        * Pingan Technology
   * Team leader
       * Jiaxun Yang
       * Student from Peking University
   * Requirements for teammates
       * College students in Beijing are preferred
       * Proficient in programming, or have an understanding of market analysis and profit models
   * Team members
      * ModestYjx，GitHub link: [ModestYjx](https://github.com/ModestYjx)

* Deatils for the project
   * Project name
       * One-stop solution for computer-assisted strawberry picking agriculture based on artificial intelligence
   * Project Introduction

>        * In recent years, strawberry picking tourism has been popular. This project aims to design an artificial intelligence-based software to realize strawberry maturity detection, crop knowledge popularization and digital agricultural management. This software will ensure that tourists can pick the most mature strawberries as soon as possible, popularize rural experience education for primary and secondary school students, so that the tourist experience is improved and the manpower employment cost is saved.

Reviewer: ```suggestion
       * In recent years, strawberry picking tourism has been popular. This project aims to design an artificial intelligence-based software to realize strawberry maturity detection, crop knowledge popularization, and digital agricultural management. This software will ensure that tourists can pick the most mature strawberries as soon as possible and that rural experience programs can be popular among primary and secondary school students. In this way, tourist experience is improved and the labor cost is saved.
```

>    * The technology fields involved in the project

Reviewer: ```suggestion
   * The technology involved in the project
```

       * Image classification
       * Graph database

### Winter is Coming

* Details of the team
   * Team name
        * Winter is Coming
   * Team leader
       * Weiming Li
       * School of Information Renmin University of China
   * Requirements for teammates

>        * Have the knowledge of raft and c++

Reviewer: ```suggestion
       * Have the knowledge of raft and C++
```

> * Deatils for the project

Reviewer: typo

   * Project name
       * raft optimization
       * Read the source code of nebula-storage to optimize the raft layer

### Linjiangxian  

* Details of the team
   * Team name
        * Linjiangxian (a famous ancient poetry in China)
   * Team leader
       * Jiankui Wang, head of Baidu Big Data Team
   * Requirements for teammates

>        * have the ability to develop mini apps and relevant development experience 

Reviewer: capitalization

> * Deatils for the project

Reviewer: typo
